src/test/java/io/pravega/connectors/hadoop/utils/SetupUtils.java

```diff
@@ -85,11 +85,17 @@ public void startAllServices(String scope) throws Exception {
                 .isInProcSegmentStore(true)
                 .segmentStoreCount(1)
                 .containerCount(4)
+                .enableTls(false)
                 // to match code change https://github.com/pravega/pravega/pull/2476
                 .keyFile("")
                 .certFile("")
+                .enableAuth(false)
                 .userName("")
                 .passwd("")
+                .passwdFile("")
```

Comment:
Why this `SetupUtils` does not populate `inProcPravegaCluster` with real config values? This is different from the FlinkConnector.

Reply:
Yes. I have found that `PravegaConfig` configurations are not used in Hadoop connectors. Created an issue to address the missing implementation https://github.com/pravega/hadoop-connectors/issues/64